contribs/gmf/src/directives/partials/drawfeature.html

```diff
@@ -90,23 +105,28 @@
 
     <div ng-switch-default>
 
+      <button
+          ng-click="efCtrl.selectedFeature = null;"
+          class="btn btn-sm btn-default">
+        <span class="fa fa-arrow-left"></span>
+        {{'List' | translate}}
+      </button>
       <div class="ngeo-drawfeature-actionbuttons">
         <button
-            ng-click="efCtrl.selectedFeature = null;"
-            class="btn btn-default">
-          {{'<- List' | translate}}
-        </button>
-        <button
-            class="btn btn-default">
+            class="btn btn-sm btn-default">
+          <span class="fa fa-mail-forward"></span>
           {{'Export' | translate}}
         </button>
         <button
             ng-click="efCtrl.features.remove(efCtrl.selectedFeature);"
-            class="btn btn-default">
+            class="btn btn-sm btn-default">
+          <span class="fa fa-trash"></span>
           {{'Delete' | translate}}
         </button>
       </div>
 
+      <div class="gmf-eol"></div>
```

Comment:
Let's go with this first. But we should replace this in the future. At least we'll try.